pandas/tests/indexes/period/test_period.py

```diff
@@ -445,17 +451,6 @@ def test_pindex_qaccess(self):
         # Todo: fix these accessors!
         assert s['05Q4'] == s[2]
 
-    def test_numpy_repeat(self):
```

Comment:
deleted since this overrides the base class implementation and appears to be doing the same thing.